README.md

```diff
@@ -90,6 +90,13 @@ summon python listEC2.py
 `python listEC2.py` is the command that summon wraps. Once the Python program exits,
 the secrets stored in temp files and in the Python process environment are gone.
 
+### Default values
+
+Default values can be set by using the `default='yourdefaultvalue'` as an addtional tag on the variable:
+```yaml
+SOME_VARIABLE_WITH_DEFAULT: !var:default='defaultvalue' path/to/variable
```

Comment:
Do you think it might help to have another example where e.g. a 'file' tag is also used might help some users? e.g. 
```
ANOTHER_VARIABLE_WITH_DEFAULT: !var:default='defaultvalue':file $env/path/to/file
```
Or would this be overkill?

Reply:
This is definitely useful as I would be unsure about how to combine tags.

Reply:
Added much more info in the readme now